packages/components/src/tabs/tablist.tsx

```diff
@@ -17,19 +17,36 @@ import type { TabListProps } from './types';
 import { useTabsContext } from './context';
 import { TabListWrapper } from './styles';
 import type { WordPressComponentProps } from '../context';
+import type { CSSProperties } from 'react';
 
 export const TabList = forwardRef<
 	HTMLDivElement,
 	WordPressComponentProps< TabListProps, 'div', false >
 >( function TabList( { children, ...otherProps }, ref ) {
 	const context = useTabsContext();
+
+	const [ indicatorPosition, setIndicatorPosition ] = useState( {
+		left: 0,
+		width: 0,
+	} );
+	const selectedId = context?.store.useState( 'selectedId' );
+	const selectedTabEl = context?.store.item( selectedId )?.element;
+
+	useEffect( () => {
+		if ( selectedTabEl )
+			setIndicatorPosition( {
+				left: selectedTabEl.offsetLeft,
```

Comment:
This also needs to work in `orientation="vertical"`.

(Not that I think the indicator styling for vertical tabs is good right now — we'll probably want to improve it at some point — but for the scope of this PR let's just make sure it isn't broken.)

Reply:
Oh, I didn't see any story for this so I assumed it wasn't officially supported. I'll take a look.

Reply:
Added.